web/src/components/countrytableoverlayifnodata.js

```diff
@@ -37,11 +33,12 @@ const CountryTableOverlayIfNoData = ({ zoneTimeIndex }) => {
 
   // TODO: Shouldn't be hardcoded
   const zonesThatCanHaveZeroProduction = ['AX', 'DK-BHM', 'CA-PE', 'ES-IB-FO'];
-  const zoneHasProductionData = (zoneData.production && Object.values(zoneData.production).every(v => v !== null)) || zonesThatCanHaveZeroProduction.includes(zoneId);
-  const isEstimated = zoneData.estimationMethod !== null;
+  const zoneHasProductionValues = zoneData.production && !Object.values(zoneData.production).every(v => v === null);
+  const zoneHasProductionData = zoneHasProductionValues || zonesThatCanHaveZeroProduction.includes(zoneId);
+  const isEstimated = !isNil(zoneData.estimationMethod);
```

Comment:
Nitpick: but shouldn't we avoid lodash here? 
```suggestion
  const isEstimated = zoneData.estimationMethod != null;
```

Reply:
We need to check for both `undefined` and `null` :)
Alternatively I considered checking if it's a string, but that also feels error-prone.

Reply:
> We need to check for both undefined and null :)

Hence `!=` and not `!==`. ;)

But why do we actually need to check both, the backend will return `null`?

Reply:
Ah, missed that part :D
`zoneData` is initially empty, so that's why it can be `undefined`... We could probably improve this, but I think that would be part of a larger revamp :)
I've changed to `!=` now.